lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -90,30 +99,15 @@ def patch_partial_renderer(klass)
       do_once(:patch_partial_renderer) do
         klass.class_eval do
```

Comment:
Could methods defined here be extracted to a module as well ?

Reply:
Same feedback as above.